### ISSUE TITLE
Incorrect documentation in nltk.stem.lancaster.LancasterStemmer class

### ISSUE THREAD

**Issue by Smeetp1234** [nltk/nltk#3179]
In [`nltk.stem.lancaster.LancasterStemmer`](https://www.nltk.org/api/nltk.stem.lancaster.html) class, there is an incorrect documentation in given example:

`from nltk.stem.lancaster import LancasterStemmer`
`st = LancasterStemmer()`
`st.stem('cement')      # ditto`

For above, given output is 'cem', but the comment # ditto signifies that output should be word 'cement' without any change.
I will request to correct the above mentioned problem, as it is showing incorrect comment as compared to given output.

**Comment by Higgs32584** [comment]
I do not believe this is an issue with documentation, as the documentation matches the output of ran code. It appears to be an issue with the lancasterStemmer, if there is one.

**Comment by jbbqqf** [comment]
Hi! Triaging older issues — the documentation/doctest is actually correct as written; the misreading is what "ditto" refers to.

Evidence:
- Reading the full docstring in `nltk/stem/lancaster.py` (lines 20–48 on `develop`), the `# ditto` comments are *not* claiming "the output is unchanged" — they refer back to the *rule comment* that introduced the previous block, e.g. `# Words starting with consonant must contain at least 3 letters and one of those letters must be a vowel` for the group `string` / `meant` / `cement`. In other words, "ditto" means "same rule as the previous example", not "same output as input".
- The doctest itself passes on `develop`: `python -m doctest nltk/stem/lancaster.py` exits 0 (so the actually-expected output for `st.stem('cement')` is `'cem'`, which matches the runtime). @Higgs32584 noted the same above — the docs match the code.
- The `cement` → `cem` behavior is also a known artifact of the Lancaster rule set itself (not a documentation bug): `cement` triggers the `-ment` stemming rule. If the concern is the stemmer's *output*, that's a separate algorithmic question rather than a doc fix.

If I'm wrong about which line is being claimed inaccurate, point me at the specific docstring line that looks wrong and I'll dig further. Otherwise this can probably be closed.

---
*Disclosure: I drafted this comment with help from Claude Code while triaging stale issues; the doctest run and docstring reading were verified manually.*

**Comment by ekaf** [comment]
Not actually a documentation bug. Thanks @jbbqqf!
